custom/icds/location_rationalization/parser.py

```diff
@@ -95,5 +98,21 @@ def _note_transition(self, operation, location_type, new_site_code, old_site_cod
             self.transitions[location_type][operation][new_site_code] = old_site_code
         elif operation == EXTRACT_OPERATION:
             self.transitions[location_type][operation][new_site_code] = old_site_code
+        self.site_codes_to_be_archived.append(old_site_code)
```

Comment:
Is it worth validating that all old site codes are different than all new site codes? I'm thinking of someone accidentally pasting a value in both the old and new columns.

Reply:
Good point.
Adding it to spec doc to cover in a different PR to keep context clear here.
Thank you!

Note: there is a check to ensure a location does not participate in two different operations, but this case is possible if the accidental copy paste in done for the same operation.

Reply:
Added under validations [here](https://docs.google.com/document/d/1h6kTr9M0FumswP3-gr_84OLeUHRmn5Y10dCpSAS3Y9g/edit#)